File: services/administration-service/src/main/java/sgptt/adminsvc/repository/entity/Academy.java
package sgptt.adminsvc.repository.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Academy {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "academy_id")
    private Long academyId;
    private String name;
}

File: services/auth-service/src/main/kotlin/com/sgpttt/authservice/repository/entity/Academy.kt
package com.sgpttt.authservice.repository.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
data class Academy(
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "academy_id")
	val academyId: Long,
	val name: String
)
